microservices/reinvent/reinvent/api_models.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import datetime
+from typing import List
+
+import pydantic
+
+
+
+
+@pydantic.dataclasses.dataclass
+class RunFineTuningJobRequest:
```

Comment:
So if this is a finetuning job request as far as I understand the protein should be passed on which we finetune, but we pass it separately, should we add it to the request instead as well?

Reply:
It is passed as a parameter to the API method, so it will be able to consume file as bytes